services/blue/src/api/DatabaseAccess/DbConnectionHolder.cs

```diff
@@ -15,6 +16,7 @@ internal class DbConnectionHolder : IDbConnectionHolder
 
         public bool HasTransaction => _transaction is not null;
 
+        [AllowNull]
```

Comment:
`[AllowNull]` is mirroring the [`IDbConnection`](https://source.dot.net/#System.Data.Common/System/Data/IDbConnection.cs,988a87787980ac79,references) signature, without it we get the warning: `[CS8767] Nullability of reference types in type of parameter doesn't match...`

`ConnectionString` won't be null when it's read, but it can be set to `null`.